test/org/zaproxy/zap/extension/pscanrulesAlpha/XPoweredByHeaderInfoLeakScannerUnitTest.java

```diff
@@ -0,0 +1,111 @@
+package org.zaproxy.zap.extension.pscanrulesAlpha;
+
+import net.htmlparser.jericho.Source;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.core.scanner.Alert;
+import org.parosproxy.paros.network.HttpMessage;
+import org.zaproxy.zap.extension.alert.ExtensionAlert;
+import org.zaproxy.zap.extension.pscan.PassiveScanThread;
+import org.zaproxy.zap.utils.I18N;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+
+/**
+ * @author Vahid Rafiei (@vahid_r)
+ * Date 2016-07-06
+ */
+
+@RunWith(MockitoJUnitRunner.class)
+public class XPoweredByHeaderInfoLeakScannerUnitTest {
```

Comment:
No problem. I'll be glad to help and learn at the same time..

But some notes:
1. These classes are not merged into zaproxy/zap-extensions repo, and only exists on your repo. So merging it would be nice.
2. They only exist on the **master** branch. What's the policy about _alpha_ and _beta_ branches? Do you want me to add this class to the alpha branch? (in pscanrulesalpha package). 
I'm not quite sure about your policy on cherry picking commits or how sync these three branches are..